applicationconfig/client/src/main/java/com/azure/applicationconfig/AzConfigCredentialsPolicy.java

```diff
@@ -38,27 +46,17 @@
     private static final String CONTENT_TYPE_HEADER = "Content-Type";
     private static final String ACCEPT_HEADER = "Accept";
 
-    private final AzConfigClient.AzConfigCredentials credentials;
-    private final HttpPipelineOptions options;
+    private final ApplicationConfigCredentials credentials;
+    private final AuthorizationHeaderProvider provider = new AuthorizationHeaderProvider();
+    private final Logger logger = LoggerFactory.getLogger(AzConfigCredentialsPolicy.class);
```

Comment:
We should chat about logging. I'm worried that we want more triggers than standard slf4j and I wonder if we create some internal libraries in azure-common for this.

Reply:
Sounds good. I wouldn't mind creating an interface ILogger in that common library and having some implementations of it so that users aren't forced to use slf4j. We can chat about it when you're free.

Reply:
Yes - and this should probably work through our azure-common library, and I'd like to bring @anuchandy into this discussion too.

Reply:
Created issue to investigate: https://github.com/Azure/azure-sdk-for-java/issues/3081